lib/process-services-cloud/src/lib/process-cloud/process-filters-cloud/process-filters-cloud.component.ts

```diff
@@ -0,0 +1,162 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, EventEmitter, Input, OnChanges, Output, SimpleChanges } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ProcessFilterCloudService } from '../services/process-filter-cloud.service';
+import { ProcessInstanceFilterRepresentationModel } from '../models/process-filter-cloud.model';
+@Component({
+    selector: 'adf-cloud-process-filters',
+    templateUrl: './process-filters-cloud.component.html',
+    styleUrls: ['process-filters-cloud.component.scss']
+})
+export class ProcessFiltersCloudComponent implements OnChanges {
+
+    /** (required) The application name */
+    @Input()
+    appName: string;
+
+    /** (optional) The filter to be selected by default */
+    @Input()
+    filterParam: ProcessInstanceFilterRepresentationModel;
```

Comment:
Wrong type it should be a model like this one
```
export class FilterParamsModel {
    id: number;
    name: string;
    index: number;

    constructor(obj?: any) {
        if (obj) {
            this.id = obj.id || null;
            this.name = obj.name || null;
            this.index = obj.index;
        }
    }
}
```
rember to change even
 `public selectFilter(newFilter: FilterParamsModel) {`